aspnetcore/data/entity-framework-6.md

```diff
@@ -17,15 +17,15 @@ This article shows how to use Entity Framework 6 in an ASP.NET Core application.
 
 To use Entity Framework 6, your project has to compile against .NET Framework, as Entity Framework 6 doesn't support .NET Core. If you need cross-platform features you will need to upgrade to [Entity Framework Core](/ef/).
```

Comment:
It's not related to this issue, but EF6 does support .NET Core now. See https://docs.microsoft.com/ef/ef6/what-is-new/#ef-630. We should address this in another PR.